packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.ts

```diff
@@ -9978,13 +9979,15 @@ interface IStickyContext {
 interface IStickyProps extends React.Props<Sticky> {
   componentRef?: IRefObject<IStickyProps>;
   isScrollSynced?: boolean;
+  onScrollStickyClassName?: string;
   stickyBackgroundColor?: string;
   stickyClassName?: string;
   stickyPosition?: StickyPositionType;
 }
 
 // @public (undocumented)
 interface IStickyState {
+  isScrollPositionNonZero: boolean | undefined;

```

Comment:
Could we simplify this to an optional state? `isScrollPositionNonZero?: boolean`  and then not define it in the constructor?